src/files/BrsFile.ts

```diff
@@ -48,7 +48,7 @@ export class BrsFile {
         this.extension = util.getExtension(this.pathAbsolute);
 
         //all BrighterScript files need to be transpiled
-        if (this.extension?.endsWith('.bs')) {
+        if (this.extension?.endsWith('.bs') || program?.options?.transpileBrs) {
             this.needsTranspiled = true;
             this.parseMode = ParseMode.BrighterScript;
```

Comment:
Is it intended to force `.brs` files to be parsed with `.bs` features? (e.g. classes and all)

Reply:
@elsassph yes it is. I am trying to support legacy code while enabling adoption of Brighterscript features. Switching all the .brs files to .bs would add a level of undesirable complexity.

Reply:
I think it needs to be mentioned in the help/doc. And maybe a few things need to be double-checked as to whether this is sufficient to fully enable brighterscript features in brs files.

Reply:
@elsassph should I add this to https://github.com/rokucommunity/brighterscript#bsconfigjson-options?
I have been using this to enable "import" and it also works with the ternary operator.

Reply:
I guess if @TwitchBronBron is ok with this compiler change he'll adjust the doc accordingly.

Oh you should add this option to the CLI as well BTW.